index.html

```diff
@@ -10,10 +10,76 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+<div class="container">
+<header>
+    <img
+        class="logo"
+        src="img/karma-logo.svg"
+        alt="Karma logo"
+    />
+    <nav>
+        <ul>
+            <li>Meet Karma</li>
+            <li>How it Works</li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+        </ul>
+    </nav>
+</header>
+<main>
+    <div class="jumbotron">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <button>Learn More</button>
+    </div>
+    <article>
+        <h2>Everyone needs a little Karma</h2>
+        <div class="icons">
+            <div class="icon">
+                <img
+                    src="img/icon-devices.svg"
+                    alt=""
+                />
+                <p class="icon-title">Internet for all devices</p>
+            </div>
+            <div class="icon">
+                <img
+                    src="img/icon-coffee.svg"
+                    alt=""
+                />
+                <p class="icon-title">Boost your productivity</p>
+            </div>
+            <div class="icon">
+                <img
+                    src="img/icon-refill.svg"
+                    alt=""
+                />
+                <p class="icon-title">Pay as You Go</p>
+            </div>
+        </div>
+    </article>
+</main>
+<footer>
+    <h3>Join us on</h3>
+    <div class="social-media
+    ">
+        <img
+            src="img/twitter-icon.svg"
+            alt="Link to Twitter"
```

Comment:
```suggestion
            alt="Twitter"
```
The `alt` description is a description of the image, which is Twitter. If this is a link then we can wrap the `img` in an `a` element